motr/linux_kernel/m0tr_main.c

```diff
@@ -64,7 +64,9 @@ module_exit(motr_exit);
  * marker is necessary to remove the warnings, keeping this blank to make
  * compiler happy.
  */
-MODULE_LICENSE();
+
+/* Added GPL per suggestions to avoid compilation error, may need to be reviewed */
+MODULE_LICENSE("GPL");
```

Comment:
This should be a temporary solution, and be fixed later.

Reply:
Solution to what problem? Just curious.